pkg/tools/assessment.go

```diff
@@ -50,9 +50,9 @@ func processResult(result *Result) error {
 		// exit with error
 		exit.Code = 2
 		exit.AddFunc(func() {
-			log.Errorf("{primary:%s} has failed - {danger:%s}", o.Tool.Name(), result.ExecuteResult.FailureMessage)
+			log.Errorf("{primary:%s} has failed - {danger:%s}", o.Tool.Name(), result.ExecuteResult.CombinedOutput)
```

Comment:
Yeah, I don't think we want to do this.  CombinedOutput is the stdout/stderr from the underlying tool.  It makes more sense to me that we do what the code is already doing, which is just push that out to stderr and then exit.

If we want to produce a better FailureMessage for a specific case we should do that.  (In the opal case look for a message from the tool and propagate that.  It's on about JSON because it's trying to parse the output as JSON and failing.)

Reply:
yes, fair point. Closing this PR, we can revisit later 👍